flytekit/core/type_engine.py

```diff
@@ -262,6 +262,8 @@ def to_literal_type(cls, python_type: Type) -> LiteralType:
 
     @classmethod
     def to_literal(cls, ctx: FlyteContext, python_val: typing.Any, python_type: Type, expected: LiteralType) -> Literal:
+        if python_val is None:
```

Comment:
@kumare3 this is okay right?  I think it was just returning a Primitive earlier with nothing set.

Reply:
I think this is right. Just that we may want a none handler?
So if python_val is None and python_type is None, then it is ok to return Void?